api/src/main/kotlin/edu/wgu/osmt/api/model/ApiCollection.kt

```diff
@@ -63,6 +63,10 @@ class ApiCollection(private val collection: Collection, private val ss: List<Ric
     val skills: List<ApiSkillSummary>
         get() = ss.map { ApiSkillSummary.fromSkill(it, appConfig) }
 
+    @get:JsonProperty
+    val owner: String?
+        get() = collection.workspaceOwner?.let { it }
```

Comment:
What does `.let { it }` get us? Should we just return collection.workspaceOwner?

Reply:
yeah it should be the same.